src/auto-evo/RunResults.cs

```diff
@@ -496,6 +496,11 @@ public List<Species> GetNewSpecies()
             return newSpecies;
         }
 
+        public NewSpeciesType? GetNewSpeciesResult(Species species)
```

Comment:
These new methods you added don't seem to be used anywhere...

Reply:
It is used, line 928. https://github.com/Revolutionary-Games/Thrive/blob/4eb6e8fd5704d1098d8ba8317ea6fc8a9b510763/src/auto-evo/RunResults.cs#L928 Originally for accessing these results from a IRunStep object, which no longer the case but thought these public methods will also be good for future uses.

Reply:
I did see that later on. Is there a reason why the species entry doesn't already have the required information?
I think these expose a bit too much of the internal workings to be public.